web/src/beta/features/PluginPlayground/Plugins/index.tsx

```diff
@@ -34,99 +34,142 @@ const Plugins: FC<Props> = ({
   updateFileTitle,
   deleteFile,
   handleFileUpload,
-  sharedPlugins
+  sharedPlugin
 }) => {
   const [isAddingNewFile, setIsAddingNewFile] = useState(false);
 
   const handleShareIconClicked = (pluginId: string): void => {
     encodeAndSharePlugin(pluginId);
   };
 
+  const customPlugin = presetPlugins.find((plugin) => plugin.id === "custom");
+  const pluginsWithoutCustom = presetPlugins.filter(
+    (plugin) => plugin.id !== "custom"
+  );
+
+  const PluginEntryItem: FC<{
+    plugin: { id: string; title: string };
+    selectedPluginId: string;
+    onSelect: (id: string) => void;
+    onShare: (id: string) => void;
+  }> = ({ plugin, selectedPluginId, onSelect, onShare }) => (
+    <EntryItem
+      key={plugin.id}
+      highlighted={selectedPluginId === plugin.id}
+      onClick={() => onSelect(plugin.id)}
+      title={plugin.title}
+      optionsMenu={[
+        {
+          id: "0",
```

Comment:
can we use id = "shared" than using 0

Reply:
Share button has been moved out to top of the panel by the latest design, there's no need to provide this option menu on plugin entry item now. Please remove this and connect the share function with the new icon button.

Reply:
Thanks. This has been removed